python/src/tokenize_sentence.rs

```diff
@@ -0,0 +1,83 @@
+// Copyright 2019 vtext developers
+//
+// Licensed under the Apache License, Version 2.0,
+// <http://apache.org/licenses/LICENSE-2.0>. This file may not be copied,
+// modified, or distributed except according to those terms.
+
+use pyo3::prelude::*;
+use pyo3::types::PyList;
+
+use vtext::tokenize::*;
```

Comment:
We shouldn't need to use any objects from there I think?

Reply:
`UnicodeSentenceTokenizer` implements `tokenize::Tokenizer` and so you have to import it into scope to be able to use it